src/dataclass_rest/http/aiohttp.py

```diff
@@ -34,7 +35,17 @@ async def _release_raw_response(self, response: ClientResponse) -> None:
 
     async def _response_body(self, response: ClientResponse) -> Any:
         try:
-            return await response.json()
+            match self.response_type:
+                case ResponseType.JSON:
+                    return await response.json()
+                case ResponseType.TEXT:
+                    return await response.text()
+                case ResponseType.BYTES:
+                    return await response.read()
+                case ResponseType.NO_CONTENT:
+                    return None
```

Comment:
should we validate content-length here?
I'd rather have 2 cases here:
* SKIP 
* NO_CONTENT